tests/test_browser.py

```diff
@@ -4660,7 +4660,7 @@ def test_access_file_after_heap_resize(self):
     self.run_browser('page.html', 'hello from file', '/report_result?15')
 
     # with separate file packager invocation
-    self.run_process([PYTHON, FILE_PACKAGER, 'data.js', '--preload', 'test.txt', '--js-output=' + 'data.js'])
+    self.run_process([PYTHON, FILE_PACKAGER, 'test.js', '--preload', 'test.txt', '--js-output=' + 'data.js'])
```

Comment:
This looks confusing, but I guess it was already confusing before. Could this just be `data.data` perhaps?

Reply:
Sure